src/objects/js/components/admin/permissions/permission-form.js

```diff
@@ -63,7 +98,7 @@ const PermissionForm = ({objectFields, dataFieldChoices, tokenChoices, objecttyp
                 name="use_fields"
                 id="id_use_fields"
                 label="Use field-based authorization"
-                disabled={mode === "read_and_write" || Object.keys(dataFieldChoices).length === 0}
+                disabled={!mode || mode === "read_and_write"}
```

Comment:
Let's also make it disabled if the objecttypes API is not available, currently you can check it despite that:

![image](https://github.com/user-attachments/assets/6134600a-1007-4e25-91bf-22c9e545cbb8)